packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -490,6 +490,15 @@ abstract final class SystemChrome {
     );
   }
 
+  static void Function(SystemUiOverlayStyle? style)? _systemUiOverlayStyleCallback;
+
+  /// Sets the callback method for responding to changes in the SystemUiOverlayStyle.
```

Comment:
This doc needs to be improved to be useful. Some tips are at https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc.

Reply:
sure. I'll done as soon as possible

Reply:
done!